packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/Providers/ModelProviders/TestData/ModelCustomizationTests/DoesNotIncludeReqCustomFieldLiteralInDefaultCtor/MockInputModel.cs

```diff
@@ -0,0 +1,9 @@
+using Microsoft.Generator.CSharp.Customization;
+
+namespace Sample.Models;
+
+public partial class MockInputModel
```

Comment:
Why does this test not customize the enum?

Reply:
ah, I overlooked this. I meant to have the original spec property as a string literal.. not an enum. This is fixed now, thank you.